transifex/native/django/management/commands/transifex.py

```diff
@@ -22,7 +23,9 @@ class Command(BaseCommand):
 
     def __init__(self, *args, **kwargs):
         super(Command, self).__init__(*args, **kwargs)
-        self.subcommands = {'push': Push(), 'migrate': Migrate()}
+        self.subcommands = {'migrate': Migrate(),
+                            'push': Push(),
+                            'try-templatetag': TryTemplatetag()}
```

Comment:
Minor: you could store the classes instead of the instances and instantiate only the command that will actually be executed.

Reply:
We have to invoke methods on these in `add_arguments` and we won't know the selected subcommand until `handle`. In any case, I don't think it's worth it.